src/main/webapp/help-projectConfig-slackTokenCredentialsId.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
This file should probably be named `help-projectConfig-slackTokenCredentialId.html` (i.e. non-plural "Credential")